src/mozanalysis/experiment.py

```diff
@@ -464,8 +458,17 @@ def _build_enrollments_query(self, time_limits, enrollments_query_type):
         """Return SQL to query a list of enrollments and their branches"""
         if enrollments_query_type == "normandy":
             return self._build_enrollments_query_normandy(time_limits)
-        elif enrollments_query_type == "fenix-fallback":
-            return self._build_enrollments_query_fenix_baseline(time_limits)
+        elif (
+            enrollments_query_type == "glean-event"
+            or enrollments_query_type == "fenix-fallback"
```

Comment:
Should we keep the "fenix-fallback" option around though for now for backwards compatibility? Is anyone else but Jetstream using this?

Reply:
Let's keep it around for interactive use for now unless it's getting in your way.

Reply:
We can keep it around and remove it later